test/ref_ops_test.cpp

```diff
@@ -953,6 +953,41 @@ TEST_CASE(concat_test)
     }
 }
 
+TEST_CASE(concat_dyn_test)
+{
+    migraphx::program p;
+    auto* mm = p.get_main_module();
+    int axis = 0;
+    migraphx::shape s0{migraphx::shape::int32_type, {{2, 4, 2}, {2, 3, 2}}};
+    migraphx::shape s1{migraphx::shape::int32_type, {{3, 4, 4}, {2, 3, 2}}};
+    migraphx::shape s2{migraphx::shape::int32_type, {{1, 5, 3}, {2, 3, 2}}};
+
+    auto input0 = mm->add_parameter("X", s0);
+    auto input1 = mm->add_parameter("Y", s1);
+    auto input2 = mm->add_parameter("Z", s2);
+    mm->add_instruction(migraphx::make_op("concat", {{"axis", axis}}), input0, input1, input2);
+    p.compile(migraphx::ref::target{});
+
+    migraphx::shape static_shape0{migraphx::shape::int32_type, {2, 2}, {2, 1}};
+    migraphx::shape static_shape1{migraphx::shape::int32_type, {3, 2}, {2, 1}};
+    migraphx::shape static_shape2{migraphx::shape::int32_type, {1, 2}, {2, 1}};
```

Comment:
These look like standard strides, do we need to specify them?

Reply:
Not strictly necessary, but it does no harm and test coverage is more robust if we don't set up every case in exactly the same way.

Reply:
not specifying strides here, shouldn't affect test coverage for this PR.  Specifying them doesn't do any harm, but my convention if  strides are  standard shape then, in MIGraphX we would let the constructor of shape handle it.

Reply:
Yea it would better to not specify them for standard shapes, so it can make clear to those reading this that it is using standard shape for the test.